packages/shorebird_cli/lib/src/validators/shorebird_flutter_validator.dart

```diff
@@ -117,6 +121,8 @@ This can cause unexpected behavior if you are switching between the tools and th
           ? await shorebirdFlutter.getVersion()
           : await shorebirdFlutter.getSystemVersion();
     } on ProcessException catch (error) {
+      if (error.errorCode == 127) throw CommandNotFoundException();
```

Comment:
It's not on the [sysexits man page](https://man.freebsd.org/cgi/man.cgi?query=sysexits) linked to in the mason_logger source, but it looks like code 127 is used by `sh` and `bash` to indicate that a command was not found on the user's PATH (https://www.gnu.org/software/bash/manual/html_node/Exit-Status.html). I was going to suggest just leaving a comment here about this, but adding this to mason_logger might not be a bad idea. Should I open a PR?